HelloWorker/hello_worker.json

```diff
@@ -24,6 +24,15 @@
       }
     },
     "localDeployment": {
+      "windows": {
```

Comment:
why do we change orders?

Reply:
I guess 90% of users are on Windows.

Reply:
The order change is here for consistency with our other docs including https://docs.improbable.io/reference/13.5/shared/worker-configuration/launch-configuration#worker-launch-configuration and https://github.com/spatialos/FlexibleProjectExample/blob/master/docs/reference/client-worker-configuration.md